cmd/main.go

```diff
@@ -434,6 +443,7 @@ func main() {
 	hostStateMonitor.Start()
 	defer hostStateMonitor.Stop()
 
+	addReleaseImagesToDBIfNeeded(db, releaseImagesArray, failOnError)
```

Comment:
Is this a problem that this runs on every pod (not just the leader)?
I assume we'll get duplicates or errors when we try to insert the same version from multiple replicas.

Maybe do this in the release syncer process?